cli/cmd/secrets.go

```diff
@@ -56,13 +56,16 @@ func createSecret() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "create [OPTIONS] SECRET [file|-]",
 		Short: "Creates a secret.",
-		Args:  cobra.ExactArgs(2),
+		Args:  cobra.RangeArgs(1, 2),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			c, err := client.New(cmd.Context())
 			if err != nil {
 				return err
 			}
-			file := args[1]
+			file := "-"
+			if len(args) == 2 {
+				file = args[1]
+			}
 			if len(file) == 0 {
 				return fmt.Errorf("Secret data source empty: %q", file)
```

Comment:
[Errors should start with lower case](https://github.com/golang/go/wiki/CodeReviewComments#error-strings)

```suggestion
				return fmt.Errorf("secret data source empty: %q", file)
```